lib/schema/documentArrayElement.js

```diff
@@ -14,18 +14,20 @@ const getConstructor = require('../helpers/discriminator/getConstructor');
  *
  * @param {String} path
  * @param {Object} options
+ * @param {Object} _schemaOptions
```

Comment:
```suggestion
 * @param {Object} schemaOptions
```

The other schematypes dont refer to it with underscore in the jsdoc when unused (re `SchemaBoolean`, `SchemaBuffer`, etc)